pkg/reconciler/serverlessservice/serverlessservice_test.go

```diff
@@ -805,6 +807,22 @@ func endpointspriv(namespace, name string, eo ...EndpointsOption) *corev1.Endpoi
 	return ep
 }
 
+// withNSubsets populates the endpoints object with numSS subsets
+// each having numAddr endpoints.
+func withNSubsets(numSS, numAddr int) EndpointsOption {
+	return func(ep *corev1.Endpoints) {
+		ep.Subsets = make([]corev1.EndpointSubset, numSS)
+		for i := 0; i < numSS; i++ {
+			ep.Subsets[i].Ports = []corev1.EndpointPort{{Port: 8012}, {Port: 8013}}
+			ep.Subsets[i].Addresses = make([]corev1.EndpointAddress, numAddr)
+			is := strconv.Itoa(i + 1)
+			for j := 0; j < numAddr; j++ {
+				ep.Subsets[i].Addresses[j].IP = fmt.Sprintf("10.1.%s.%d", is, j+1)
```

Comment:
Any reason to not do:
```suggestion
			for j := 0; j < numAddr; j++ {
				ep.Subsets[i].Addresses[j].IP = fmt.Sprintf("10.1.%d.%d", i+1, j+1)
```